fala/apps/adviser/forms.py

```diff
@@ -26,16 +28,30 @@ def __init__(self, attrs={}):
 class AdviserSearchForm(forms.Form):
     page = forms.IntegerField(required=False, widget=forms.HiddenInput())
 
-    postcode = forms.CharField(
-        label=_("Enter postcode"),
-        max_length=30,
-        help_text=_("For example, <span class='notranslate' translate='no'>SW1H 9AJ</span>"),
+    if settings.FEATURE_FLAG_NO_MAP:
```

Comment:
@exonian so this is only an if/else feature flag statement on the `postcode` form filed (change to name and helptext). So I think this already covers your request to keep it maintainable?? 

Might look like more has changed, as I did some formatting on how the form fields were laid out in this class, to keep them consistent.

Reply:
Yeah all I mean is that instead of both parts of the conditional writing out the whole field definition, which is several lines of which most are the same for both, it's possible to write the field once and then the conditional only changes the two lines it needs to change.
For me, that feels more maintainable (if we were to keep this code long term), because it's a lot easier to see what the `if` statement is doing if things like max_length and widget aren't in both the `if` and the `else` clauses.
It would look something like:
`
postcode = forms.CharField(
    label=_("Enter postcode"),
    max_length=30,
    help_text=_("For example, <span class='notranslate' translate='no'>SW1H 9AJ</span>"),
    required=False,
    widget=FalaTextInput(),
)

def __init__(self, *args, **kwargs):
    super()
    if not settings.FEATURE_FLAG_NO_MAP:
        self.fields['postcode'].label = _("Postcode")
        self.fields['postcode'].help_text=_(mark_safe("For example, <span class='notranslate' translate='no'>SW1H</span>"))
`

I don't mind us not, here, just wanted to point out this way is available and will sometimes be what we need. To my eye, it's neater, but it lacks simplicity so I get it.

Reply:
@exonian This makes sense. Have now changed.